docs/install/ubuntu-debian.rst

```diff
@@ -39,6 +39,8 @@ From the command line: ``sudo apt-get remove kolibri``.
 Upgrade
 *******
 
+.. warning:: Make sure that Kolibri is not running before upgrading.
```

Comment:
This warning is unnecessary. The .deb will restarts Kolibri as part of upgrading. If it doesn't work, it's a bug... but in no way a documentation thing.

We cannot IMO in any consistent way require the user to switch off Kolibri before upgrading.

Reply:
> If it doesn't work, it's a bug... but in no way a documentation thing.

Can't it be both?

We've seen many bugs lately related to corrupted DBs and simultaneous access. Until we're confident those are all addressed, this seems like reasonable advice.

Perhaps:

```
.. warning:: We recommend making sure Kolibri is not running before upgrading.
```

Reply:
I don't think such a warning looks very good with no further reasoning?

CC: @jredrejo with regards to the way that Kolibri is stopped and changes to the systemd service, timeouts etc... is there a chance that Kolibri will not be properly stopped?

Call is here:
https://github.com/learningequality/kolibri-installer-debian/blob/master/debian/kolibri.scripts-common#L390

Implementation is here, noting that it will ignore if stopping the system service fails (presumably to ignore if it wasn't running in the first place)
https://github.com/learningequality/kolibri-installer-debian/blob/master/debian/kolibri.scripts-common#L154

Reply:
As discussed, let's keep the warning but also open a high level issue that we may link in the warning, at least in the Debian section.